graspologic/nominate/VNviaSGM.py

```diff
@@ -288,18 +297,17 @@ def fit(
         # explanation
         self.n_seeds_ = len(close_seeds)
         seeds_fin = np.arange(self.n_seeds_)
```

Comment:
On seeds, are they really required for the vertex nomination? I thought these can be optional.

Reply:
you are correct, but as written this class was doing the seeded version. i believe this will be overhauled/replaced at some point, so im just not going to address in this PR